openfeature/provider-java/src/main/scala/io/cardell/openfeature/provider/java/JavaConverters.scala

```diff
@@ -66,7 +66,7 @@ object ToJavaConverters {
 
   def evaluationContext(ec: EvaluationContext): JContext = {
     val values = ec.values.map { case (k, v) => (k, contextValue(v)) }.asJava
-    new JContext(values)
+    ec.targetingKey.fold(new JContext(values))(tk => new JContext(tk, values))
```

Comment:
An alternative would be to add the targetingKey beforehand to `values`. This might be more efficient as the `JContext` constructor creates a new `HashMap` instance if a targetingKey is passed, but probably negligible as context shouldn't have that many keys.